charts/vault-operator/README.md

```diff
@@ -34,10 +34,13 @@ helm upgrade --install vault-operator banzaicloud-stable/vault-operator
 
 To install the chart backed with a cluster-wide Etcd Operator, use the following:
 
+**Please note that the etcd-operator has been deprecated for a long time (chart and code as well), and the last version of the chart is not installable on Kubernetes 1.22 and onwards. We don't offer any kind of support for etcd-operator.**
```

Comment:
Actually this is a more recently maintained project if you need it: https://github.com/cilium/cilium-etcd-operator.

Reply:
Hi, I removed etcd from chart dependencies and we are not using it for our setup.
I also reported a bug regarding etcd.